src/style.css

```
@@ -1 +1,12 @@
-/* styles go here */
+.label {
+  display: block;
+  margin-bottom: 8px;
```

```suggestion
  margin-bottom: 10px;
```
if I remember correctly